content/guides/docker-concepts/getting-started/get-docker.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Get Docker 
```

Comment:
We also have a Get Docker topic at https://docs.docker.com/get-docker/. I believe having two topics with the same title would probably cannibalize SEO. Can we rename this topic? Additionally, as most of the download content is repeated in https://docs.docker.com/get-docker/, we should also look at reusing content instead of duplicating.

Reply:
The video title is "Get Docker Desktop", and this page talks exclusively about DD, so maybe "Get Docker Desktop" would work?

Reply:
@dvdksn That's a good suggestion. I will modify the title to "Get Docker Desktop" instead.